src/core/variable.jl

```diff
@@ -762,6 +762,40 @@ function variable_mc_switch_state(pm::AbstractUnbalancedPowerModel; nw::Int=nw_i
     report && _IM.sol_component_value(pm, pmd_it_sym, nw, :switch, :state, ids(pm, nw, :switch_dispatchable), state)
 end
 
+"capacitor (with capcontrol) switch state (open/close) variables"
```

Comment:
should correct to follow new conventions for docstring format that include the function signature